tests/lib/rules/lines-around-comment.js

```diff
@@ -0,0 +1,131 @@
+            code: "bar()\n// line line line\n\nvar a = 1;",
+            args: [2, { afterLineComment: true }]
+        },
+        {
+            code: "foo()\n\n// line line line\nvar a = 1;",
+            args: [2, { beforeLineComment: true }]
+        },
+        {
+            code: "foo()\n\n// line line line\n\nvar a = 1;",
+            args: [2, { beforeLineComment: true, afterLineComment: true }]
+        },
+        {
+            code: "foo()\n\n// line line line\n// line line\n\nvar a = 1;",
+            args: [2, { beforeLineComment: true, afterLineComment: true }]
+        },
+
+        // block comments
+        {
+            code: "bar()\n\n/** A Block comment with a an empty line after\n *\n */\nvar a = 1;",
+            args: [2, { afterBlockComment: false, beforeBlockComment: true }]
+        },
+        {
+            code: "bar()\n/** block block block\n * block \n */\nvar a = 1;",
+            args: [2, { afterBlockComment: false }]
+        },
+        {
+            code: "bar()\n\n/** block block block\n * block \n */\n\nvar a = 1;",
+            args: [2, { afterBlockComment: true, beforeBlockComment: true }]
+        },
+
+        // inline comments (should not ever warn)
+        {
+            code: "foo() // An inline comment with a an empty line after\nvar a = 1;",
+            args: [2, { afterLineComment: true, beforeLineComment: true }]
+        },
+        {
+            code: "foo();\nbar() /* An inline block comment with a an empty line after\n *\n */\nvar a = 1;",
+            args: [2, { beforeBlockComment: true }]
+        },
+
+        // mixed comment (some block & some line)
+        {
+            code: "bar()\n/** block block block\n * block \n */\n//line line line\nvar a = 1;",
+            args: [2, { afterBlockComment: true }]
+        },
+        {
+            code: "bar()\n/** block block block\n * block \n */\n//line line line\nvar a = 1;",
+            args: [2, { beforeLineComment: true }]
+        }
+
+    ],
+
+    invalid: [
+
+        // default rules
+        {
+            code: "bar()\n/** block block block\n * block \n */\nvar a = 1;",
+            errors: [{ message: afterMessage, type: "Block" }]
+        },
+
+        // line comments
+        {
+            code: "baz()\n// A line comment with no empty line after\nvar a = 1;",
+            args: [2, { afterLineComment: true }],
+            errors: [{ message: afterMessage, type: "Line" }]
+        },
+        {
+            code: "baz()\n// A line comment with no empty line after\nvar a = 1;",
+            args: [2, { beforeLineComment: true, afterLineComment: false }],
+            errors: [{ message: beforeMessage, type: "Line" }]
+        },
+        {
+            code: "baz()\n// A line comment with no empty line after\nvar a = 1;",
+            args: [2, { beforeLineComment: true, afterLineComment: true }],
+            errors: [{ message: beforeMessage, type: "Line" }, { message: afterMessage, type: "Line" }]
+        },
```

Comment:
Can you add the following test case with `beforeLineComment: true`.

```js
bar(); // comment
// another comment
```

I would expect one error in line `2`.